stories/toolbar/toolbar.stories.js

```diff
@@ -99,7 +99,7 @@ export const overflow = () => `<div style="height:250px">
         <button class="fd-button fd-button--compact fd-button--transparent">Save</button>
         <button class="fd-button fd-button--compact fd-button--transparent">Copy</button>
         <span class="fd-toolbar__separator"></span>
-        <div class="fd-popover fd-toolbar__overflow">
+        <div class="fd-popover fd-toolbar__overflow-button">
```

Comment:
how is `fd-toolbar__overflow-button` a button when it's applied on a div?

Reply:
Modified.